gradle/verification-metadata.xml

```diff
@@ -3180,31 +3180,36 @@
             <sha256 value="a82ac5bc24bcbf6ba9eb70f334d6782e25245c8da36d9848ad553b5b7b68efd1" origin="Generated by Gradle"/>
          </artifact>
       </component>
-      <component group="org.bouncycastle" name="bcpkix-jdk15on" version="1.64">
-         <artifact name="bcpkix-jdk15on-1.64.jar">
-            <sha256 value="84669138b1d99143e2c009024f67824ab8d3edb9b05b7591f5ebfb020a4bda71" origin="Generated by Gradle"/>
+      <component group="org.bouncycastle" name="bcpkix-jdk18on" version="1.77">
+         <artifact name="bcpkix-jdk18on-1.77.jar">
+            <sha256 value="1ac7fe8efd5b2f38cdc165be5a0675734fe44808dab92707201f03a535d6f1b8" origin="Generated by Gradle"/>
          </artifact>
       </component>
       <component group="org.bouncycastle" name="bcprov-jdk15on" version="1.60">
          <artifact name="bcprov-jdk15on-1.60.jar">
             <sha256 value="7f1a0e6badab38666f8467a9a0ee96656b2f8ec8623867ed34f3cdc173b7ee07" origin="Generated by Gradle"/>
          </artifact>
       </component>
-      <component group="org.bouncycastle" name="bcprov-jdk15on" version="1.64">
-         <artifact name="bcprov-jdk15on-1.64.jar">
-            <sha256 value="a4f463ce552b908a722fa198ef4892a226b3225e453f8df10d5c0a5bfe5db6b6" origin="Generated by Gradle"/>
-         </artifact>
-      </component>
       <component group="org.bouncycastle" name="bcprov-jdk15on" version="1.69">
```

Comment:
I've double checked this and indeed these old dependencies are only used in test fixture and build tools:

![image](https://github.com/elastic/elasticsearch/assets/4357326/9b2d0937-de7f-42ea-9233-0a2cb156056d)

![image](https://github.com/elastic/elasticsearch/assets/4357326/d4652970-3857-443d-a909-b84f7a3d44b3)